xbmc/music/infoscanner/MusicInfoScanner.cpp

```diff
@@ -715,6 +715,8 @@ void CMusicInfoScanner::FileItemsToAlbums(CFileItemList& items, VECALBUMS& album
       {
         std::string strJoinPhrase = (it == --common.end() ? "" : g_advancedSettings.m_musicItemSeparator);
         CArtistCredit artistCredit(*it, strJoinPhrase);
+        if (it->compare(g_localizeStrings.Get(340)) == 0)
```

Comment:
Not really, the variable that it is comparing against is set a few lines above in the same function so the localized string will always be the same.